nx/lib/nx/lin_alg.ex

```diff
@@ -1160,7 +1181,7 @@ defmodule Nx.LinAlg do
 
```

Comment:
The Nx.sum calls here are facing the same problem as before, in which Torchx can't return {:u, 64}, and the defn compiler enforces that check. A simple solution is converting the boolean tensors to {:s, 8}, but this would imply that Nx-core is aware of Torchx's limitations somehow.

Any ideas?

cc @seanmor5 @josevalim

Reply:
> the defn compiler enforces that check

Do you mean the evaluator is raising when we get a different value from Torchx? Ping me later and we can pair ono it!